libraries/stdlib/jvm/src/kotlin/internal/PlatformImplementations.kt

```diff
@@ -51,20 +52,28 @@ internal val IMPLEMENTATIONS: PlatformImplementations = run {
     val version = getJavaVersion()
     if (version >= 0x10008) {
         try {
-            return@run castToBaseType<PlatformImplementations>(Class.forName("kotlin.internal.jdk8.JDK8PlatformImplementations").newInstance())
-        } catch (e: ClassNotFoundException) { }
+            return@run castToBaseType<PlatformImplementations>(
+                Class.forName("kotlin.internal.jdk8.JDK8PlatformImplementations").newInstance()
+            )
+        } catch (e: ClassNotFoundException) {
+        }
```

Comment:
Please revert formatting changes unrelated to the code regions you have changed.